azure-pipelines.yml

```diff
@@ -20,6 +20,7 @@ pr:
       - dco.txt
       - .github/ISSUE_TEMPLATE/*.md
       - .github/labeler.yml
```

Comment:
Is this path still necessary to be listed here? Same with `github/workflows/*`?

Reply:
they don't hurt and once github actions Prs comes in they wont use precious CI :)